### PR TITLE
Optimizer disaster recovery docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added, or deleted? (Required)
Optimizer disaster recovery docs

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/1513
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/pd-recover.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ When you use `pd-recover` to recover the PD cluster, you need to specify `alloc-
 
 3. Go back to the window where the `port-forward` command is executed, press <kbd>Ctrl</kbd>+<kbd>C</kbd> to stop and exit.
```
Comment (hfxsd) [inline en/pd-recover.md:150]:
```suggestion
3. Go back to the window where the `port-forward` command is executed, and then press <kbd>Ctrl</kbd>+<kbd>C</kbd> to stop and exit.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: en/pd-recover.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ kubectl get tc test -n test -o='go-template={{.status.clusterID}}{{"\n"}}'
 6821434242797747735
 ```
 
-### Get Alloc ID
+### Step 2: Get Alloc ID
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply en/pd-recover.md:52]: 中文用的是冒号，其他文档里好像也用的冒号。在 TiDB Operator 文档里大家似乎比较认同冒号。。。

**Pull request merged**